### DISCUSSION TITLE
Webhost doesn't support Postgres... workaround?

### DISCUSSION THREAD

**Discussion by jehtee** [immich-app/immich#9809] [Q&A]
I want to install this on Dreamhost.  However Dreamhost does not support Postgres on their shared hosting plans.  

Is there any workaround for this?  

Or is the only solution to pay for a plan that support Postgres, find a different host, or find a different photo management solution?

Thanks!

**Comment by mmomjian** [comment]
Do they support Docker at all..? I don't understand how a hosting company could specifically paywall *Postgres* if they allow Docker.

Either way, it's impossible to run Immich without Postgres, somewhere. In theory that could be on another server using a mesh VPN (latency will be poor).

**Reply by jehtee** [reply]
Yes I think they support Docker, but yeah that workaround seems suboptimal.  Thanks for the fast response.

**Reply by mmomjian** [reply]
If they support Docker how can they stop you from running a Postgres docker container? Are they reading your docker compose files? I don't really understand.

**Reply by jehtee** [reply]
I will have to ask them for more details.  They just explained to me that Postgres is not supported on their shared hosting plans... that would require an upgrade.

**Reply by mmomjian** [reply]
They probably just mean that they won't provide a Postgres server for you. If you have Docker, you can run basically whatever you want.

**Reply by jehtee** [reply]
Ok, so Dreamhost doesn't actually provide Postgres OR docker on shared plans.  So that's that.
